packages/twenty-server/src/engine/api/graphql/graphql-query-runner/graphql-query-parsers/graphql-query-selected-fields/graphql-selected-fields.parser.ts

```diff
@@ -33,12 +33,18 @@ export class GraphqlQuerySelectedFieldsParser {
       relations: {},
     };
 
+    const hasEdges = Object.keys(graphqlSelectedFields).includes('edges');
```

Comment:
I had checked that with @Weiko, it is not to take into account queried fields outside of edges